README.md

```diff
@@ -49,55 +50,65 @@ pip install git+https://github.com/keras-team/keras-cv.git tensorflow --upgrade
 ## Quickstart
 
 ```python
-import keras_cv
 import tensorflow as tf
 from tensorflow import keras
+from keras_cv.layers import RandomFlip, RandAugment, CutMix, Mixup
+from keras_cv.models import ResNet50V2Backbone, ImageClassifier
 import tensorflow_datasets as tfds
 
-# Create a preprocessing pipeline
+# Create a preprocessing pipeline with augmentations
+BATCH_SIZE = 16
+NUM_CLASSES = 3
 augmenter = keras.Sequential(
-    layers=[
-        keras_cv.layers.RandomFlip(),
-        keras_cv.layers.RandAugment(value_range=(0, 255)),
-        keras_cv.layers.CutMix(),
-        keras_cv.layers.MixUp()
+    [
+        RandomFlip(),
+        RandAugment(value_range=(0, 255)),
+        CutMix(),
+        MixUp()
     ]
 )
 
 def preprocess_data(images, labels, augment=False):
-    labels = tf.one_hot(labels, 3)
+    labels = tf.one_hot(labels, NUM_CLASSES)
     inputs = {"images": images, "labels": labels}
     outputs = augmenter(inputs) if augment else inputs
     return outputs['images'], outputs['labels']
 
-# Augment a `tf.data.Dataset`
 train_dataset, test_dataset = tfds.load(
     'rock_paper_scissors',
     as_supervised=True,
     split=['train', 'test'],
 )
-train_dataset = train_dataset.batch(16).map(
+train_dataset = train_dataset.batch(BATCH_SIZE).map(
     lambda x, y: preprocess_data(x, y, augment=True),
         num_parallel_calls=tf.data.AUTOTUNE).prefetch(
             tf.data.AUTOTUNE)
-test_dataset = test_dataset.batch(16).map(
+test_dataset = test_dataset.batch(BATCH_SIZE).map(
     preprocess_data, num_parallel_calls=tf.data.AUTOTUNE).prefetch(
         tf.data.AUTOTUNE)
 
-# Create a model
-densenet = keras_cv.models.DenseNet121(
-    include_rescaling=True,
-    include_top=True,
-    num_classes=3
+# Create a model using a pretrained backbone
+backbone = ResNet50V2Backbone.from_preset("resnet50_v2_imagenet")
+model = ImageClassifier(
```

Comment:
As this read-me file is the first thing a visitor will make impression on keras-cv, it might be needed to consider adding advanced modeling, i.e. object detection (2d, 3d), segmentation. Maybe adding individual readme to corresponding [modeling directories](https://github.com/keras-team/keras-cv/tree/master/keras_cv/models); just a thoughts.

Reply:
Eventually we will have a "getting started" guide with a tour of the library, but for now I added a link to our developer guides above